### PR TITLE
CAL-237 Fixed offset in image chip page

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
This PR fixes the Image Chip Offset (10 pixels off).
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@rzwiefel @glenhein 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@andrewkfiedler 
#### How should this be tested?
Build / Install / Ingest NITF / Chip the NITF and test top/bottom edge cases
#### What are the relevant tickets?
[CAL-237](https://codice.atlassian.net/browse/CAL-237)
#### Checklist:
- [ ] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/imaging/imaging-actionprovider-chip/src/main/webapp/js/chipping.js [hunk 1]
```diff
@@ -53,7 +52,7 @@ function draw() {
 function mouseMove(e) {
     if (drag) {
         rect.w = (e.pageX - this.offsetLeft) - rect.startX;
-        rect.h = (e.pageY - this.offsetTop) - rect.startY;
+        rect.h = (e.pageY - this.offsetTop) - rect.startY  - parseInt($(".chip-title").css("margin-bottom"), 10);
```
Comment (glenhein) [inline catalog/imaging/imaging-actionprovider-chip/src/main/webapp/js/chipping.js]: "10" looks like a magic number. Can you make it a constant?

**Comment by jlcsmith** [comment]
Built and tested - looks good.

File: catalog/imaging/imaging-actionprovider-chip/src/main/webapp/js/chipping.js (hunk 1 above)
Comment (bdeining) [inline catalog/imaging/imaging-actionprovider-chip/src/main/webapp/js/chipping.js]: done
